ethcore/Cargo.toml

```diff
@@ -61,6 +61,7 @@ rlp = "0.4.0"
 rlp_derive = { path = "../util/rlp-derive" }
 rustc-hex = "1.0"
 serde = "1.0"
+serde_json = "1.0"
```

Comment:
seems it's only for tests, can you move it to `dev-dependencies`?

Reply:
Yes, I will do that.